src/components/chartArea/chartArea.js

```diff
@@ -166,9 +168,25 @@ class ChartArea extends React.Component {
 
     return (
       <div ref={this.containerRef}>
-        <Chart width={chartWidth} {...chartProps}>
+        <Chart
+          width={chartWidth}
+          {...chartProps}
+          legendData={legendData}
+          legendOrientation="horizontal"
+          legendPosition="bottom"
+          padding={{
```

Comment:
padding is necessary in order to prevent the legend from getting arbitrarily clipped. Kind of a bummer we have to do this right now. Prefer to see that go to PF in the future...but for now... wrapper it is.

Reply:
got it!